tests/modeladmin/tests.py

```diff
@@ -615,6 +617,49 @@ def test_log_actions(self):
         self.assertEqual(ma.log_change(mock_request, self.band, 'changed'), LogEntry.objects.latest('id'))
         self.assertEqual(ma.log_change(mock_request, self.band, 'deleted'), LogEntry.objects.latest('id'))
 
+    @mock.patch('django.contrib.admin.options.get_deleted_objects')
+    def test_get_deleted_objects_called(self, mock_delobjs):
+        # the get_deleted_objects() function imported to
+        # django.contrib.admin.options from django.contrib.admin.utils is
+        # called by modeladmin's get_deleted_objects() method (#17962).
+        ma = ModelAdmin(Band, self.site)
+        mock_request = MockRequest()
+        mock_request.user = User.objects.create(username='bob')
+        ma.get_deleted_objects(
```

Comment:
Single line here is fine (as the style guide says, we allow up to 119 characters if it improve readability).